### ISSUE TITLE
Better docker integration

### ISSUE THREAD

**Issue by sjlevine** [sjlevine/atom-slime#29]
May need to expose host, port, etc.

**Comment by haz** [comment]
Feel free to assign this to me and remove the "help wanted" label.

**Comment by daewok** [comment]
Something to think about with this is that when running in a docker image, M-. will return file names that are meaningless to Atom.

**Comment by haz** [comment]
@daewok -- aye, that's why the local (host) machine needs to have symlinks setup to mirror that of what docker sees. The custom setup we have in mind should be in our private wiki...

**Comment by daewok** [comment]
That would work for GNU/Linux users (but would probably be annoying to set up). What about Windows users, with their silly drive letters?

**Comment by haz** [comment]
They would simply need to find a different set of instructions ;). Perhaps the issue is misnamed -- we don't want to do anything fully general for docker integration into atom; rather we just want to expose those settings necessary for those using MERS-site instructions to follow along with minimal effort.

**Comment by daewok** [comment]
Fair enough, but it'll still be an eventual issue for this project I feel, even outside of Docker integration. For people running production code, it's not uncommon to connect to a remote swank server to do debugging, and the filesystems are likely different there as well. I'd advocate adding to the atom-slime roadmap some hooks for translating between local and remote file names. (For reference, see `slime-to-lisp-filename-function` and `slime-from-lisp-filename-function`).

**Comment by haz** [comment]
Brilliant idea. I'd suggest we keep this issue open even after pull request #38 goes through.
